2dcontext/tools/gentestutils.py

```diff
@@ -166,14 +166,14 @@ def backref_html(name):
         return ''.join(backrefs)
 
     def make_flat_image(filename, w, h, r,g,b,a):
-        if os.path.exists('%s/%s' % (IMAGEOUTPUTDIR, filename)):
-            return filename
-        surface = cairo.ImageSurface(cairo.FORMAT_ARGB32, w, h)
-        cr = cairo.Context(surface)
-        cr.set_source_rgba(r, g, b, a)
-        cr.rectangle(0, 0, w, h)
-        cr.fill()
-        surface.write_to_png('%s/%s' % (IMAGEOUTPUTDIR, filename))
+        # if os.path.exists('%s/%s' % (IMAGEOUTPUTDIR, filename)):
+        #     return filename
+        # surface = cairo.ImageSurface(cairo.FORMAT_ARGB32, w, h)
+        # cr = cairo.Context(surface)
+        # cr.set_source_rgba(r, g, b, a)
+        # cr.rectangle(0, 0, w, h)
+        # cr.fill()
+        # surface.write_to_png('%s/%s' % (IMAGEOUTPUTDIR, filename))
```

Comment:
This doesn't seem correct.

Reply:
Indeed. This is a mistake (brought about by #19780)